### ISSUE TITLE
BNS import usability: support `export-data.tar.gz` file standalone

### ISSUE THREAD

**Issue by zone117x** [blockstack/stacks-blockchain-api#393]

**Comment by agraebe** [comment]
is this still relevant? any more context here we could work with?

**Comment by zone117x** [comment]
Yeah I think so, it's referring to the BNS import process which requires manually copying in several of these txt files https://github.com/blockstack/stacks-blockchain/tree/master/stx-genesis 

It should work with the single export-data.tar.gz file that contains these.

**Comment by zone117x** [comment]
Update:

At the moment, the `v1-import` data is only used for BNS, and is optional. If the export files are not specified, the API will run without full BNS support. This is _probably_ the right setup for BNS subdomains, because the import process is lengthy, the subdomain dataset files are large, and many entities running an API, like exchanges, likely don't need any BNS support.

However, this issue https://github.com/blockstack/stacks-blockchain-api/issues/423 provides important locked STX data from the Stacks 1.0 crowdsale, and it requires importing the 18MB [`chainstate.txt`](https://github.com/blockstack/stacks-blockchain/blob/master/stx-genesis/chainstate.txt) file.

This will no longer make the `v1-import` step completely optional. It needs modified so that importing chainstate.txt is a separate (and required) step, while the BNS subdomain files can still be optional.

Additionally, this repo (or at the very least the docker image) should be able to perform the chainstate.txt import without needing the separate step of downloading and configuring it. The stacks-node repo also has this requirement, and solves it by committing chainstate.txt to git, then using a rust build-time script to compress the file and bundle it inside the executable. I think the API repo should have a similar setup.

**Comment by agraebe** [comment]
the linked PR is merged, I assume this can be closed? reopen if I'm wrong
